### ISSUE TITLE
Cannot find module 'esm'

### ISSUE THREAD

**Issue by baboon-king** [pixari/vue-i18n-extract#128]
# Cannot find module 'esm'

## system info

```shell
  System:
    OS: macOS 12.0
    CPU: (8) arm64 Apple M1
    Memory: 71.44 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 15.14.0 - ~/.nvm/versions/node/v15.14.0/bin/node
    npm: 7.10.0 - ~/.nvm/versions/node/v15.14.0/bin/npm
  Browsers:
    Chrome Canary: 93.0.4570.0
    Safari: 15.0

```

## package.json

```json
{
  "name": "xxxxx.form.web",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint",
    "i18n:report": "vue-cli-service i18n:report --src \"./src/**/*.?(js|vue)\" --locales \"./src/locales/**/*.json\""
  },
  "dependencies": {
    "axios": "^0.18.0",
    "canvas-compress": "^1.1.5",
    "core-js": "^3.6.5",
    "crypto-js": "^3.1.9-1",
    "dayjs": "^1.10.5",
    "element-ui": "^2.13.2",
    "font-awesome": "^4.7.0",
    "js-cookie": "^2.2.1",
    "oidc-client": "^1.10.1",
    "pdfjs-dist": "2.5.207",
    "qrcodejs2": "0.0.2",
    "qs": "^6.5.2",
    "vue": "^2.6.11",
    "vue-i18n": "^8.22.3",
    "vue-json-views": "^0.1.1",
    "vue-pdf": "^4.1.0",
    "vue-quill-editor": "^3.0.6",
    "vue-router": "^3.2.0",
    "vuex": "^3.4.0",
    "vuex-oidc": "^3.4.0",
    "wangeditor": "^3.1.1",
    "xlsx": "^0.14.3"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~4.5.0",
    "@vue/cli-plugin-eslint": "~4.5.0",
    "@vue/cli-plugin-router": "~4.5.0",
    "@vue/cli-plugin-vuex": "~4.5.0",
    "@vue/cli-service": "~4.5.0",
    "@vue/eslint-config-prettier": "^6.0.0",
    "babel-eslint": "^10.1.0",
    "eslint": "^6.7.2",
    "eslint-plugin-prettier": "^3.3.1",
    "eslint-plugin-vue": "^6.2.2",
    "lint-staged": "^9.5.0",
    "prettier": "^2.2.1",
    "sass": "^1.26.5",
    "sass-loader": "^8.0.2",
    "vue-cli-plugin-i18n": "~2.1.1",
    "vue-template-compiler": "^2.6.11"
  },
  "gitHooks": {
    "pre-commit": "lint-staged"
  },
  "lint-staged": {
    "*.{js,jsx,vue}": ["vue-cli-service lint", "git add"]
  }
}
```

## stack

```shell
xxxx@xxxxxdeAir adc.form.web % npm run serve

> adc.form.web@0.1.0 serve
> vue-cli-service serve

 ERROR  Error: Cannot find module 'esm'
Require stack:
- /Users/bbbbb/adc/adc.form.web/node_modules/vue-i18n-extract/dist-node/index.js
- /Users/bbbbb/adc/adc.form.web/node_modules/vue-cli-plugin-i18n/report.js
- /Users/bbbbb/adc/adc.form.web/node_modules/vue-cli-plugin-i18n/index.js
- /Users/bbbbb/adc/adc.form.web/node_modules/@vue/cli-service/lib/Service.js
- /Users/bbbbb/adc/adc.form.web/node_modules/@vue/cli-service/bin/vue-cli-service.js
Error: Cannot find module 'esm'
Require stack:
- /Users/bbbbb/adc/adc.form.web/node_modules/vue-i18n-extract/dist-node/index.js
- /Users/bbbbb/adc/adc.form.web/node_modules/vue-cli-plugin-i18n/report.js
- /Users/bbbbb/adc/adc.form.web/node_modules/vue-cli-plugin-i18n/index.js
- /Users/bbbbb/adc/adc.form.web/node_modules/@vue/cli-service/lib/Service.js
- /Users/bbbbb/adc/adc.form.web/node_modules/@vue/cli-service/bin/vue-cli-service.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/Users/bbbbb/adc/adc.form.web/node_modules/vue-i18n-extract/dist-node/index.js:49:11)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:996:19)
```

**Comment by pixari** [comment]
Hi! Thank you for reporting this.
I'll have a look at it as soon as possible.

**Comment by Spittal** [comment]
I believe this is solved as of 2.0.0. Let us know if you still have issues.
